spec/presenters/submission_presenter_spec.rb

```diff
@@ -85,6 +75,7 @@
   end
 
   describe "#created_at" do
+    let(:rates) { double }
```

Comment:
If you have `rates` in so many `describe`s maybe it's good idea to have it on the very upper level?